coverage/hare_coverage.py

```diff
@@ -0,0 +1,300 @@
+from abc import ABC, abstractmethod

```

Comment:
Please add a license, as this is a new file.
Thanks!

Reply:
Done